spec/util/vmdb-logger_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'util/vmdb-logger'
 require 'util/miq-password'
+require 'active_support/core_ext/string'
```

Comment:
Think you need this in `lib/gems/pending/util/vmdb-logger.rb` too

Reply:
Preferably in the vmdb-logger and *not* in the spec.

Reply:
Oh I see, so `active_support/core_ext/string` should be included via `util/vmdb-logger`? @Fryguy

Reply:
@fryguy all done.

Reply:
Yes, since that's where you actually use it (for the truncate call)